examples/notebooks/piTopDisplay.ipynb

```diff
@@ -4,180 +4,38 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Using hte piTop[4] display"
+    "# Using the piTop[4] display"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": 1,
    "metadata": {},
    "source": [
-    "#r \"nuget:pi-top\"\r\n",
-    "#r \"nuget:pi-top.FoundationKit\"\r\n",
+    "#i \"nuget:/home/pi/localNuget\""
```

Comment:
this line is not required as the source should be available in the user nuget config